tests/test_io.py

```diff
@@ -45,15 +51,15 @@ def test_read_write_avro(randomdata, datadir_local):
 def _write_schema(data, path):
     schema = {"type": "record", "name": "testschema"}
     schema["fields"] = [{"name": str(c), "type": "float"} for c in data.columns.values]
-    with open(path, 'wb') as f:
+    with open(path, 'w') as f:
```

Comment:
Why is this being changed from `wb` to `w` here, but `w` to `wb` elsewhere?

Reply:
This schema is written in a text format but was (I think incorrectly) marked as binary. Almost everything else blocks writes is binary format (the actual data) but in 2 it didn't matter which way the file was opened.